src/taipy/core/task/task.py

```diff
@@ -178,7 +178,7 @@ def submit(
         force: bool = False,
         wait: bool = False,
         timeout: Optional[Union[float, int]] = None,
-    ):
+    ) -> "Job":
```

Comment:
I am not sure we can do that. 
Each Job references a Task. So the Task class cannot reference the Job class without a circular import.

Reply:
There is no import at this point...